feat(tem): scale plan MTA-5622 (#4576)
scaleway/docs-content: pages/transactional-email/how-to/manage-tem-plans.mdx

---
meta:
  title: How to manage TEM plans
  description: Guide to managing your TEM plans with Scaleway, including upgrading, downgrading, and monitoring usage.
content:
  h1: How to manage TEM plans
  paragraph: Step-by-step guide to managing your TEM plans with Scaleway.
tags: tem plans management transactional email
dates:
  validation: 2024-03-12
  posted: 2024-03-12
categories:
  - transactional-email
---

> Plans are Transactional Email tiers of service. Each plan includes a set of features, pricing, and usage limits tailored to meet your use cases, ranging from low-volume occasional senders to high-volume enterprise users. Plans are scoped to Projects.

Author: ```suggestion
Plans are the service tiers for Transactional Email, each offering specific features, pricing, and usage limits to suit different needs—from occasional low-volume senders to high-volume enterprise users. Plans are scoped to Projects.
```

---
meta:
  title: How to manage TEM plans
  description: Guide to managing your TEM plans with Scaleway, including upgrading, downgrading, and monitoring usage.
content:
  h1: How to manage TEM plans
  paragraph: Step-by-step guide to managing your TEM plans with Scaleway.
tags: tem plans management transactional email
dates:
  validation: 2024-03-12
  posted: 2024-03-12
categories:
  - transactional-email
---

> Plans are Transaction Email tiers of service. Each plan includes a set of features, pricing, and usage limits tailored to meet your use cases, ranging from low-volume occasional senders to high-volume enterprise users. Plans are scoped to Projects.

Reviewer: ```suggestion
Plans are Transactional Email tiers of service. Each plan includes a set of features, pricing, and usage limits tailored to meet your use cases, ranging from low-volume occasional senders to high-volume enterprise users. Plans are scoped to Projects.
```

> Plans are Transaction Email tiers of service. Each plan includes a set of features, pricing, and usage limits tailored to meet your use cases, ranging from low-volume occasional senders to high-volume enterprise users. Plans are scoped to Projects.

Reviewer 2: ```suggestion
Plans are the service tiers for Transactional Email, each offering specific features, pricing, and usage limits to suit different needs—from occasional low-volume senders to high-volume enterprise users. Plans are scoped to Projects.
```

Two plans are available:
  - **Essential** - pay-as-you-go plan designed for occasional or low-volume email sending needs. This plan offers simplified management and a single webhook per domain.
  - **Scale** - fixed-price plan tailored for high-volume email sending. Includes 100K emails. Additional emails are charged. This plan includes a dedicated IP address and unlimited webhooks per domain. It also features advanced blocklist management, allowing for manual additions.

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization

> - [Configured your domain(s) with Transactional Email](/transactional-email/how-to/configure-domain-with-transactional-email/)
> - Added [SPF, DKIM](/transactional-email/how-to/add-spf-dkim-records-to-your-domain/), [MX](/transactional-email/how-to/add-mx-record-to-your-domain/) and [DMARC](/transactional-email/how-to/add-dmarc-record/) records to your domain(s)

Reviewer 3: ```suggestion
```
A domain is not needed to configure a Plan (in fact, a plan is chosen before any domain creation)

    </Message>
5. Type **UPGRADE** to confirm.

## How to keep track of your resources

Each TEM plan has a number of domains and webhooks included in the plan. To keep track of your usage and consumption, you can follow the steps below:

1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. Your domains display.
2. Click the **Plans** tab.
3. Scroll to the **Current plan** section. This section shows you how many domains and webhooks are currently being used in your Project.

    If you reach or surpass the free capacity of your plan, the number of resources will be indicated in red.

    <Lightbox src="scaleway-tem-current-plan.webp" alt="A screenshot of the Scaleway console shows the current plan section. The Project is subscribed to the essential plan. 3 domains are used in this Project. A maximum of 5 domains are available with the Essential plan. 2 Webhooks used in this Project. 1 webhook is available per domain with the Essential plan. Free emails per month: 0/300. A button to upgrade plan is also shown." />

>     You can click the information tooltip next to the number to see the detail of how much each resource was surpassed beyond the included amount.

Reviewer 2: ```suggestion
    You can click the information tooltip next to the number to view details on how much each resource exceeded the included amount.
```
